pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,195 @@
+//+build linux
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"net"
+	"os"
+
+	"github.com/ti-mo/conntrack"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/util/sysctl"
+)
+
+var _ ConnTrack = new(connTrack)
```

Comment:
Please add a comment for these anonymous `var` declarations as suggested in an earlier comment

Reply:
As these compile-time checks are recommended by effective-go (https://golang.org/doc/effective_go.html) and some best practices, could we leave them as it is without comment? Adding a comment at every check may become tedious and repetitive. Any particular reason that you think why comments are useful? Thanks.

Reply:
I wouldn't say they are strictly recommended:
> By convention, such declarations are only used when there are no static conversions already present in the code, which is a rare event.

I haven't looked at the code in details so I don't know if this is the case here or not for all your interfaces.

I did like the K8s approach of leaving a comment, but I see we don't necessarily do it in the rest of the Antrea code either, so I won't push for this too hard :)

Reply:
I added comment where interface and struct names are different.